scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -229,11 +247,18 @@ export const generateRelatedAssetIndex = async () => {
   const generatedAssetData: AssetsById = JSON.parse(
     await fs.promises.readFile(generatedAssetsPath, 'utf8'),
   )
-  const relatedAssetIndex: Record<AssetId, AssetId[]> = {}
-  const assetDataWithRelatedAssetKeys: AssetsById = { ...generatedAssetData }
+  const relatedAssetIndex: Record<AssetId, AssetId[]> = JSON.parse(
```

Comment:
The other crux of this PR - now that we skip generation for known assets (unless `rebuildAll` is explicitly set to true), we can't just initialize this to an empty object, else we'd end up with an empty `relatedAssetIndex.json` as a result.